scripts/kvm-test.py

```diff
@@ -99,6 +100,11 @@ def __init__(self, args):
         self.iso = f'/tmp/kvm-test/{self.release}-test.iso'
         self.hostname = f'{self.release}-test'
         self.target = f'/tmp/kvm-test/{self.hostname}.img'
+        parent = pathlib.Path(self.target).parent
+        self.ovmf = {
+                'CODE': parent / 'OVMF_CODE_4M_ms.fd',
```

Comment:
Maybe prefix with the hostname in case we're spinning two vms at the same time?

Reply:
Ah good point. I hadn't thought of doing that. I wonder how confusing that would have been.